superset/assets/javascripts/explore/stores/getLanguage.js

```diff
@@ -0,0 +1,24 @@
+
+const $ = window.$ = require('jquery');
+
+export function getLanguage() {
+  function getLocale() {
+    const locale = $.ajax({
+      url: '/superset/rest/api/getLocale',
+      async: false,
+    });
+    return JSON.parse(locale.responseText).language;
+  }
+  switch (getLocale()) {
```

Comment:
I don't see the problem, your default value for the language is already 'en'. Anyway this needs a more thoroughly review from someone at airbnb.